CCDFDataModel/CProj4ToCF.cpp

```diff
@@ -447,16 +460,19 @@ int CProj4ToCF::convertProjToCF( CDF::Variable *projectionVariable, const char *
   return 0;
 }
 
-
 int CProj4ToCF::convertCFToProj( CDF::Variable *projectionVariable,CT::string *proj4String){
+    
   proj4String->copy("Unsupported projection");
   try{
     CT::string grid_mapping_name;
     projectionVariable->getAttribute("grid_mapping_name")->getDataAsString(&grid_mapping_name);
+    CREPORT_INFO_NODOC(CT::string("Unsupported projection. Using CF conventions to determine projection. grid_mapping_name variable: ")+grid_mapping_name,
```

Comment:
Rephrase to something like: "Unable to get projection from proj4 string, trying  to use CF convention to determine projection. grid_mapping variable : " etc.